Tools/AP_Bootloader/board_types.txt

```diff
@@ -416,7 +416,7 @@ AP_HW_CUAV-7-NANO                    7000
 AP_HW_VUAV-V7pro                     7100
 
 # please fill gaps in the above ranges rather than adding past ID #7109
-
+AP_HW_F4BY_F427                        1530
```

Comment:
move up in file so as to be in proper order of numbers

Reply:
> move up in file so as to be in proper order of numbers

I ran rebase to my path, now I need to wait again for the tests to finish. we take too long, this is causing problems.
It’s good that no one has taken my ID yet.

Reply:
> move up in file so as to be in proper order of numbers

done, moved from line 420 to 335